app/src/main/java/com/example/composetemplate/presentation/common/LectureCard.kt

```diff
@@ -0,0 +1,144 @@
+package com.example.composetemplate.presentation.common
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.composetemplate.R
+import com.example.composetemplate.data.models.server_models.Lecture
+import com.example.composetemplate.ui.theme.CustomTheme
+import com.example.composetemplate.utils.Constants.Companion.LECTURE_CARD_PAST_TIME
+import com.example.composetemplate.utils.DateUtil
+
+@Composable
+fun LectureCard(
+    modifier: Modifier = Modifier,
+    lecture: Lecture,
+    isExpanded: Boolean = false,
+) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(if (isExpanded) 300.dp else 150.dp)
+            .padding(8.dp)
+            .clickable { },
+        shape = RoundedCornerShape(0.dp, 13.dp, 13.dp, 13.dp),
+        colors = CardDefaults.cardColors(containerColor = CustomTheme.colors.loginEnable)
+    ) {
+        Column(
+            horizontalAlignment = Alignment.Start,
+            modifier = modifier
+                .fillMaxSize()
+                .padding(8.dp)
+        ) {
+            if (isExpanded) {
+                Text(
+                    text = "Next Lecture",
```

Comment:
Please move this to constants. It will make the config strings easier to use.

Reply:
your right I will do it